addon/lib/field.js

```diff
@@ -102,16 +101,7 @@ export default EmberObject.extend(Evented, {
             question: { slug: this._question.slug },
             [camelize(__typename.replace(/Answer$/, "Value"))]: null
           },
-          { document: this.document, field: this },
-          this._answer && Array.isArray(this._answer.value)
```

Comment:
Would this still allow to get the case state (valid, etc.) after building the document tree?

Meaning, will we still catch invalid inputs in table rows?